oeps/oep-0001.rst

```diff
@@ -0,0 +1,318 @@
+see which numbers have already been assigned.
+
+.. _central OEP repository: http://github.com/edx/open-edx-proposals
+
+As updates are necessary, the OEP author can update the pull request.
+
+OEPs can be reference pull requests to other Open edX repositiories which
+will act as reference implementations.
+
+OEP Review & Resolution
+-----------------------
+
+Once the authors have completed an OEP, they may request a review from
+OEP reviewers. These reviewers will select one member to act as the final
+arbiter on the OEP. This arbiter will be recorded in the "Arbiter"
+header on the OEP. The remaining reviewers and the rest of the Open edX
+community will be given the opportunity to comment on the OEP, with the
+Arbiter serving to keep the discussion on track and to evaluate when
+it has reached a final conclusion.
+
+For an OEP to be accepted by the Arbiter, it must meet certain minimum
+criteria. It must be a clear and complete description of the proposed
+enhancement. The enhancement must represent a net improvement. The proposed
+implementation, if applicable, must pass the existing Open edX review process.
+
+An OEP can be assigned the status "Deferred". The OEP author or Arbiter can
+assign the OEP this status when no progress is being made on the OEP. Once a
+OEP is deferred, the OEP Author can re-assign it to draft status.
+
+An OEP can also be "Rejected" by the Arbiter. Perhaps after all is said and
+done it was not a good idea. It is still important to have a record of this
+fact. The "Withdrawn" status is similar - it means that the OEP author
+themselves has decided that the OEP is actually a bad idea, or has accepted
+that a competing proposal is a better alternative.
+
+When a OEP is Accepted, Rejected or Withdrawn, the OEP should be updated
+accordingly. In addition to updating the status field, at the very least the
+Resolution header should be added with a link to the relevant post in the edx-code
+mailing list archive or to the appropriate section of the PR.
+
+OEPs can also be superseded by a different OEP, rendering the original
+obsolete.
+
+The possible paths of the status of OEPs are as follows:
+
+.. image:: oep-0001/state-flow.png
+
+Please note that OEP statuses do not necessary coincide with the status of
+the pull request that contains the OEP. For example, OEPs that have been "Rejected"
+should still be merged, but should be marked as Rejected. This preserves the rationale
+and description of the OEP in the generated documentation.
+
+Likewise, an OEP that is in "Draft" may be merged to capture a set of edits,
+and to make the proposal more visible to community comment. From that point, additional
+pull requests can be opened to edit the "Draft" OEP, until it converges to being
+either "Accepted" or "Rejected".
+
+OEP Roles
+---------
+
+Each OEP will have an Arbiter (as described in `OEP Review & Resolution`_). The
+Arbiter will be chosen by Eddie Fagin (edX's Chief Archictect). The Arbiter
```

Comment:
I think we should state this as "will be chosen by the edX Chief Architect (currently Eddie Fagin)".  Not to presuppose Eddie's departure, but to assign the task to the role rather than the person.